src/partials/device_profile_required.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="card">
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo" />
+  </div>
+  <div id="spinnerId" style="display:block">
+    <img class="spinner" src="{{spinner}}" alt="" />
```

Comment:
Any reason not to use the spinner handlebar here? Should just be able to use the handle bar markup like so:
```  
{{> spinner}}
```

Reply:
Because I didn't realize that we could just use that... will be updated in the next commit